pyproject.toml

```diff
@@ -1,6 +1,75 @@
 [build-system]
-requires = ["setuptools >=42.0"]
-build-backend = "setuptools.build_meta"
+requires = ["flit-core >= 3.3"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "build"
+version = "0.7.0"
+description = "A simple, correct PEP 517 build frontend"
+readme = "README.md"
+requires-python = ">=3.6"
+license.file = "LICENSE"
+authors = [
```

Comment:
We should fill out the maintainer's list too, no?

Reply:
Added - please let me know if you'd like a different name or email being used.

Reply:
Looks good, ming fixing the merge conflicts?

Reply:
Also I think Filipe is still maintainer too :+1:

Reply:
> Looks good, ming fixing the merge conflicts?

Will do after 0.8.0 is out, otherwise I'd have to do it again for the changelog.

> Also I think Filipe is still maintainer too 👍

If it's ok with Filipe I'd just rather list everyone under authors. Everyone's got different ideas about what the difference between author and maintainer is and CM advises that maintainer should only be used if the original author is no longer active.